packages/formstr-app/package.json

```diff
@@ -13,6 +13,7 @@
     "@types/react": "^18.2.23",
     "@types/react-dom": "^18.2.8",
     "antd": "^5.11.2",
+    "classnames": "^2.5.1",
```

Comment:
what does this do?

Reply:
helps in writing clean code while applying classnames based on conditions.
refer Asterisk classname

Reply:
removed package